build/.goreleaser.yml

```diff
@@ -0,0 +1,90 @@
+builds:
+  - id: nri-vsphere-nix
+    main: ./cmd/nri-vsphere
+    binary: nri-vsphere
+    ldflags:
+      - -s -w -X main.integrationVersion={{.Version}} -X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: 386
+
+  - id: nri-vsphere-win
+    main: ./cmd/nri-vsphere
+    binary: nri-vsphere
+    ldflags:
+      - -s -w -X main.integrationVersion={{.Version}} -X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    goos:
+      - windows
+    goarch:
+      - 386
+      - amd64
+    hooks:
+      pre: build/windows/set_exe_properties.sh {{ .Env.TAG }} "vsphere"
+
+nfpms:
+  - id: linux
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}-1_{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure vsphere Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from vsphere components."
+    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
+
+    dependencies:
+      - "newrelic-infra"
+
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+
+    contents:
+      - src: "vsphere-config.yml.sample"
+        dst: "/etc/newrelic-infra/integrations.d/vsphere-config.yml.sample"
+        type: config
+      - src: "vsphere-performance.metrics"
```

Comment:
All other pipelines have two sections, one for rpm and the other for deb, any reason here we were able to include both in one?

Reply:
it seems better, but I am not sure if it is exactly the same

Reply:
I saw some pipelines with them separated on other pipelines as well. I think the main reason is to be able to separate the naming scheme bor the packages. I think this was because of legacy reasons, so perhaps newer integrations do not need that.

I'd doublecheck in the repo just to be sure that the naming scheme has not change.

Reply:
based on [this one ](https://github.com/newrelic/nri-mysql/blob/master/build/.goreleaser.yml#L34)
I will double check rpm. I teasted deb and is ok

Reply:
rpm packages are ok